packages/phpstan-static-type-mapper/tests/TypeMapper/ArrayTypeMapperTest.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Rector\PHPStanStaticTypeMapper\TypeMapper;
+namespace Rector\PHPStanStaticTypeMapper\Tests\TypeMapper;
```

Comment:
@dobrry The namespace was incorrect here in your original PR and it caused few troubles. I guess PHPStan should tell you.

Each test has to be in `/Tests/` namespace. Maybe we should cover it with PHPStan rule?

Reply:
Weird, but I do remember that I added the Test in the namespace. I checked how it looks like in other packages to make it the same. Maybe I screw it somehow and some point. Before pushing the changes I ran the PHPStan and I had only one issue that I see only on my local environment.

It is clear that the tests should have their own namespace but PHPStan could be helpful to avoid such problems.

Reply:
That happens
https://github.com/rectorphp/rector/pull/4054/files#diff-12672d608506733ec39087a8afdea004